aflr/api_resources/voice.py

```diff
@@ -1,18 +1,11 @@
-import aflr
-from aflr.api_request import APIRequest
+from aflr.helper_classes import ListableResource
 
 
-class Voice(APIRequest):
+class Voice(ListableResource):
     OBJECT_NAME = "voice"
+    resource_path = "/voice"
+    list_parameters_path = "voice/parameter"
```

Comment:
hmm why dont this have a backslash at the beginning

Reply:
it has now